tests/legacy-cli/e2e/tests/basic/update-7.0.ts

```diff
@@ -0,0 +1,53 @@
+import { createProjectFromAsset } from '../../utils/assets';
+import { expectFileToMatch, expectFileToExist } from '../../utils/fs';
+import { ng, silentNpm } from '../../utils/process';
+import {
+  isPrereleaseCli, removeHttpDep, useBuiltPackages, useCIChrome, useCIDefaults,
+} from '../../utils/project';
+import { expectToFail } from '../../utils/utils';
+
+
+export default async function () {
+  const extraUpdateArgs = await isPrereleaseCli() ? ['--next', '--force'] : [];
+
+  // Create new project from previous version files.
+  // We must use the original NPM packages to force a real update.
+  // This also means that we can only truly test updates after a release.
+  await createProjectFromAsset('7.0-project', true);
+  // Update the CLI.
+  await ng('update', '@angular/cli', ...extraUpdateArgs);
+  // Should update the lazy route syntax via update-lazy-module-paths.
+  await expectFileToMatch('src/app/app-routing.module.ts',
+    `loadChildren: () => import('./lazy/lazy.module').then(m => m.LazyModule)`);
+  // Unnecessary es6 polyfills should be removed via drop-es6-polyfills.
+  await expectToFail(() => expectFileToMatch('src/polyfills.ts', `import 'core-js/es6/symbol';`));
+  await expectToFail(() => expectFileToMatch('src/polyfills.ts', `import 'core-js/es6/set';`));
+  // Should update tsconfig and src/browserslist via differential-loading.ts
+  await expectFileToMatch('tsconfig.json', `"target": "es2015",`);
+  await expectFileToMatch('src/browserslist', `\nChrome 41`);
```

Comment:
Blocked on https://github.com/angular/angular-cli/pull/14226 and https://github.com/angular/angular-cli/issues/14232.

Reply:
IS this RC-blocking? or could we just remove this assertion from the test and fix for RC1?

Reply:
I *think* it might prevent diff. loading from working properly. But am not sure. There are already PRs for both of those problems though.

Reply:
It won't affect differential loading working properly.  However, its presence will guarantee every new application performs differential loading.
Chrome 41 does not support ES2015 but is required by the Google search bot.  If an application is publicly available and wishes to be fully indexed, it needs to support Chrome 41.  For applications that are intranet only or accessed via authentication, this is not needed.